### DISCUSSION TITLE
Incomprehensible behavior with replacement ExpandNodeId in "_node_to_string" method in "xmlexporter.py"

### DISCUSSION THREAD

**Discussion by xydan83** [FreeOpcUa/opcua-asyncio#673] [Q&A] [answered]
Hi. Help me, plz, in one question.
![image](https://user-images.githubusercontent.com/86729310/135113621-707a1b2a-da96-4ab7-812f-0397f51d5ed2.png)
Suppose, I create such a node `nsu=http://some.server.namespace;i=1`, I ignore "ns", because according to the specification, if we have "nsu" specified, then we ignore "ns".
But this code replaces **ExpandNodeId** with a regular **NodeId**, so I lose "nsu" and even "srv".
Maybe I misunderstood?

**Comment by oroulet** [comment, marked as answer]
importer and exporter are full of bugs.  a PR is more than welcome
